rlberry/agents/bandits/randomized_agents.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+from rlberry.agents.bandits import BanditWithSimplePolicy
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class RandomizedAgent(BanditWithSimplePolicy):
+    """
+    Agent for bandit environment using randomized policy like EXP3.
+
+    Parameters
+    -----------
+    env : rlberry bandit environment
+        See :class:`~rlberry.envs.bandits.Bandit`.
+
+    index_function : callable or None, default = None
+        Compute the index for an arm using the past rewards and sampling
+        probability on this arm and the current time t.
+        If None, use loss-based importance weighted estimator.
+
+    prob_function : callable or None, default = None
+        Compute the sampling probability for an arm using its index.
+        If None, use exponential softmax probabilities with any learning rate.
```

Comment:
I don't understand the bit about "with any learning rate", can you explain more. Maybe you can say "with learning rate from ..." and cite an article.

Reply:
It's just bad writing from me. I'll resubmit (need to make a small change to the learning rate definition actually -- there are quite a few in the literature, I'm trying to pick a somewhat standard one).